### ISSUE TITLE
Profile template/component doesn't have configurable alt text for headshots

### ISSUE THREAD

**Issue by adamjohnson** [wvuweb/wvu-design-system#279]
![Profile template screenshot](https://user-images.githubusercontent.com/575865/89547368-aab0b480-d7d3-11ea-8a9d-10c35fc59bb8.png)

## Steps to reproduce the issue

  1. Create a profile page and fill out all the requisite fields.
  1. Be sure to fill in the alt text in Page Properties > Custom Data.

## Results

The profile image has a hardcoded alt text value of `thumbnail_alt`. You can see this in `views/components/_wvu-profile.html` [lines 41 and 45](https://github.com/wvuweb/wvu-design-system/blob/master/views/components/_wvu-profile.html#L41-L45).

## Expected results

Instead of the alt text value being `thumbnail_alt`, it should be `<r:page:data name="thumbnail_alt" />`.

For easy copy paste:

```html
alt="<r:page:data name="thumbnail_alt" />"
```
